setup.cfg

```diff
@@ -1,13 +1,3 @@
-[check-manifest]
-ignore =
-    */test
-    */test/*
-    .*
-    h/browser
-    h/browser/*
-    node_modules
-    tox.ini
-
```

Comment:
In case it's not clear -- I've removed the `check-manifest` Travis job and config because moving everything into `src/` makes it next to impossible to screw up the `MANIFEST.in` file as it can now just say `graft src` and everything is included.